core/src/main/scala/org/apache/spark/ui/JettyUtils.scala

```diff
@@ -496,10 +496,16 @@ private[spark] case class ServerInfo(
   }
 
   def stop(): Unit = {
+    val threadPool = server.getThreadPool
```

Comment:
How about setting the idel timeout after 
```
server.stop()
```
Is there any difference?

Reply:
then there will be a chance that we trigger the bug before setting the idle timeout?

Reply:
Ah, I see. This is a bit complex..